File: Bars.GkhRF/Entities/Payment/Payment.cs
namespace Bars.GkhRf.Entities
{
    using Bars.Gkh.Entities;

    /// <summary>
    /// Оплата
    /// </summary>
    public class Payment : BaseGkhEntity
    {
        /// <summary>
        /// Жилой дом
        /// </summary>
        public virtual RealityObject RealityObject { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? ChargePopulationCr { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? ChargePopulationHireRf { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? ChargePopulationCr185 { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? ChargePopulationBldRepair { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? PaidPopulationCr { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? PaidPopulationHireRf { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? PaidPopulationCr185 { get; set; }

        /// <summary>
        /// Не хранимое
        /// </summary>
        public virtual decimal? PaidPopulationBldRepair { get; set; }
    }
}

File: Bars.GkhRF/libs/B4/model/Payment.js
Ext.define('B4.model.Payment', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'Payment'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'RealityObject', defaultValue: null },
        { name: 'Address' },
        { name: 'Municipality' },
        { name: 'ChargePopulationCr' },
        { name: 'PaidPopulationCr' },
        { name: 'ChargePopulationHireRf' },
        { name: 'PaidPopulationHireRf' },
        { name: 'ChargePopulationCr185' },
        { name: 'PaidPopulationCr185' },
        { name: 'ChargePopulationBldRepair' },
        { name: 'PaidPopulationBldRepair' }
    ]
});